index.html

```diff
@@ -0,0 +1,174 @@
+          <iframe class="bali-spirit-story-video" 
+                  src="https://www.youtube.com/embed/9pJIwR5O5_M" 
+                  title="YouTube video player" 
+                  frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>  
+          </iframe> <!-- Video From Bali Shakti Sundari youtube channel -->
+        </div>
+      </section>
+      <section class="previous-events">
+        <h3 class="heading--secondary underline">Previous events</h3>
+        <div class="paragraph-box">
+          <p class="paragraph-normal">In 2008 the idea was born to create a global community gathering of cultures coming together in Bali for inspiration to make the world a better place. </p> <!-- Content from Bali spirit festival website -->
+        </div>
+        <div class="image-container">
+          <div class="image-box">
+            <div class="image-overlay"></div>
+            <div class="overlay-text">
+              <h3 class="heading--secondary">2021</h3>
+              <p class="paragraph-normal">Nusa Penida</p>
+            </div>
+            
+            <img src="assets/images/about-page-pic-1.jpg" alt="Picture of Balinese dancers"> 
+            <!--- Photo by Aditya Agarwal: https://www.pexels.com/photo/two-women-dancing-while-wearing-dresses-at-night-time-1477310/-->
+          </div>
+          <div class="image-box">
+            <div class="image-overlay"></div>
+            <div class="overlay-text">
+              <h3 class="heading--secondary">2021</h3>
+              <p class="paragraph-normal">Canggu</p>
+            </div>
+            
+            <img src="assets/images/about-page-pic-2.jpg" alt="Picture of Balinese dancers"> 
+            <!-- Photo by Wendy Wei from Pexels -->
+          </div>
+        </div>
+      </section>
+    </div>
+  </template>
+  
+  <!-- Mobile menu template -->
+  <template class="js-template-mobile-menu">
+    <div class="mobile-menu">
+      <img class="close-icon" src="assets/icons/close.svg">
+      <ul class="mobile-menu__items js-menu">
+        <li><a class="js-home-page-link" href="#">Home</a></li>
+        <li><a class="js-about-page-link" href="#">About</a></li>
+      </ul>
+    </div>
+  </template>
+
+  <!-- Speakers template -->
+  <template class="js-template-speakers">
+      <div class="featured-speakers__card">
+        <div class="featured-speakers__img">
+          <img class="js-speaker-pic" src="" alt="Image of speaker"> <!-- Placehoder images from https://i.pravatar.cc-->
+        </div>
+        <div class="featured-speakers__card__details">
+          <h3 class="heading--secondary js-speaker-name"></h3>
+          <h5 class="heading--secondary underline js-speaker-specialty"></h5>
+          <p class="paragraph-normal js-speaker-desription">With decade of experience in yoga, Dewi will guide you through this journay</p>
+        </div>
+      </div>
+  </template>
+
```

Comment:
- [ ] Kindly add the footer of the homepage and the about page to the respective page

Reply:
Addressed ✔